tests/MediaTest.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+namespace RonasIT\Media\Tests;
+
+use App\Models\User;
+use RonasIT\Media\Models\Media;
+use App\Tests\Support\MediaTestTrait;
+use Illuminate\Http\Testing\File;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use RonasIT\Support\Traits\FilesUploadTrait;
+
+class MediaTest extends ModuleTestCase
+{
+    use FilesUploadTrait;
+    use MediaTestTrait;
+
+    protected $admin;
+    protected $user;
+    protected File $file;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->admin = User::find(1);
+        $this->user = User::find(2);
+        $this->file = UploadedFile::fake()->image('file.png', 600, 600);
+    }
+
+    public function testCreate(): void
+    {
+        $this->mockGenerateFilename();
+
+        $response = $this->actingAs($this->admin)->json('post', '/media', ['file' => $this->file]);
+
+        $response->assertCreated();
```

Comment:
please assert response content for each `200` code responses

Reply:
Deleted the tests because it was decided to do them in another merge request.